tests/config.yaml

```diff
@@ -1,12 +1,12 @@
 document: modularity-testing
 version: 1
 name: docker.io/modularitycontainers/mariadb
-modulemd-url: http://pkgs.fedoraproject.org/cgit/modules/mariadb.git/plain/mariadb.yaml?h=f26
+modulemd-url: https://src.fedoraproject.org/modules/mariadb/raw/master/f/mariadb.yaml
 service:
     port: 3036
 module:
     docker:
-        start: "docker run -it --name mariadb_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306"
```

Comment:
tests were failing for me b/c MTF didn't not remove the containers during testing and docker complained about "container with such name already exists", let me try again